java/app/src/main/res/drawable/add_to_google_wallet_button_background.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <size android:minWidth="300px" android:minHeight="48px"/>
```

Comment:
Make sure to [always use `dp` on Android](https://developer.android.com/training/multiscreen/screendensities), which are screen density independent units. Using px will naturally make elements be drawn in smaller sizes on screens with a larger pixel density.

Reply:
Updated to use DP.